backend/docker-compose.yml

```diff
@@ -1,7 +1,52 @@
-version: '3'volumes:
-
+version: '3.4'
```

Comment:
Specifying version as of 1.27.0 (2020-09-07) is not required

Reply:
Thanks, fixed